meetings/serializers.py

```diff
@@ -51,9 +50,33 @@ def validate_title(self, data):
             )
         return data
 
+    def validate(self, data):
+        room = data['room']
+        check_in = data['date_from']
+        check_out = data['date_to']
+        users = data['users']
+
+        check_room_time = Reservation.objects.filter(room=room).filter(
+            Q(date_from__lte=check_in, date_to__gte=check_in) |
+            Q(date_from__lte=check_out, date_to__gte=check_out) |
+            Q(date_from__gte=check_in, date_to__lte=check_out)
```

Comment:
Let's put into a single expression instead of two `filter` statements, I mean put `room=room` expression inside this filter